spec/features/dance_comments/4_user_updates_spec.rb

```diff
@@ -67,6 +67,8 @@
     click_button "Update"
 
     expect(page).to have_content("Comment can't be blank")
+
+    # CANNOT SEEM TO TEST FOR TYPE CAN'T BE BLANK.
   end
```

Comment:
Hmm, you might have to use `unselect` here. Requires knowing which option is selected. 
http://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FNode%2FActions%3Aunselect